prow/scripts/provision-vm-and-test-reconciler-with-cli-k3d.sh

```diff
@@ -144,7 +156,13 @@ log::info "Copying Kyma to the instance"
 #shellcheck disable=SC2088
 utils::compress_send_to_vm "${ZONE}" "kyma-integration-test-${RANDOM_ID}" "/home/prow/go/src/github.com/kyma-project/kyma" "~/kyma"
 
-log::info "Triggering the installation"
-gcloud compute ssh --ssh-key-file="${SSH_KEY_FILE_PATH:-/root/.ssh/user/google_compute_engine}" --verbosity="${GCLOUD_SSH_LOG_LEVEL:-error}" --quiet --zone="${ZONE}" --command="sudo bash" --ssh-flag="-o ServerAliveInterval=30" "kyma-integration-test-${RANDOM_ID}" < "${SCRIPT_DIR}/cluster-integration/reconciler-integration-with-cli-k3d.sh"
+# run the relevant script to deploy Kyma and run fast-integration tests
+if [[ "${TEST_KYMA_UPGRADE}" == "true" ]]; then
```

Comment:
Does it make sense to check if `KYMA_UPGRADE_SOURCE` is not empty to activate the upgrade scenario? If yes, then also let's remove the `TEST_KYMA_UPGRADE` env-var completely.